client/extensions/woocommerce/app/dashboard/setup-tasks.js

```diff
@@ -1,92 +1,114 @@
 /**
  * External dependencies
  */
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
 import { localize } from 'i18n-calypso';
 import React, { Component, PropTypes } from 'react';
 
 /**
  * Internal dependencies
  */
+import {
+	areSetupChoicesLoading,
+	getOptedOutOfShippingSetup,
+	getOptedOutofTaxesSetup,
+	getTriedCustomizerDuringInitialSetup
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/15223/commits/004863a3c5f48ce79a2d0ea93ba459112536c873